packages/documentation-site/patternfly-docs/content/get-started/upgrade.md

```diff
@@ -1,105 +1,162 @@
 
-This guide outlines the steps required to upgrade your codebase to PatternFly 6 and provides additional resources to support your migration efforts.  
+In the following sections, this upgrade guide outlines steps required to migrate your codebase to PatternFly 6 and additional resources to support your journey:
 
-**Note:** If your product uses a custom solution to replicate PatternFly styling (without using PatternFly components), then it will need to be re-skinned. We recognize that this may be a large undertaking, so we encourage you to reach out to the PatternFly team so that we can help support this work. 
+1. [Notable changes](#notable-changes): Significant changes you should be aware of before starting your upgrade.
+1. [Upgrade steps](#how-to-upgrade): Step-by-step instructions for upgrading.
+1. [Help resources](#get-help): Easy-access links to get help from the PatternFly team.
 
-## Get help 
+<Divider />
+
+## Notable changes
+
+Before you upgrade, familiarize yourself with these significant changes. While some are handled by codemods, some changes *could* require manual intervention and should be kept in mind as you upgrade.
+
+### Components
+
+1. [Chip](/components/chip): Replaced with [label](/components/label).
+1. [Tile](/components/tile): Replaced with [card](components/card).
+1. [Text](https://v5-archive.patternfly.org/components/text): Renamed to ["content"](/components/content).  
+1. [Expandable section](/components/expandable-section): Removed `isActive`.
+1. [Empty state](/components/empty-state): Refactored. 
+    - This refactoring is not *entirely* handled by codemods. Depending on your implementation, your empty states could also require manual updates. For more details, search "empty state" in the [release notes table](/get-started/upgrade/release-notes). 
 
-If you need support as you upgrade, the PatternFly team is here to help! 
+### Charts
 
-Don't hesitate to reach out to us on [Slack](https://join.slack.com/t/patternfly/shared_invite/zt-1npmqswgk-bF2R1E2rglV8jz5DNTezMQ) or ask a question on [our GitHub discussion board](https://github.com/orgs/patternfly/discussions). Since this upgrade includes significant visual changes, please contact us if you have a question about styles or if you think something looks wrong in your UI. We'll always do our best to answer your questions, double-check your work, and connect you with the right people quickly. 
+To support multiple chart libraries, Victory-based charts have moved to a "victory" directory in [@patternfly/react-charts](https://www.npmjs.com/package/@patternfly/react-charts). Your import paths will need to align with this change. 
+
+- **Old path:** `import { Area } from '@patternfly/react-charts';` 
+- **New path:** `import { Area } from '@patternfly/react-charts/victory';` 
+
+To install Victory packages, you can either: 
+- Install the single "victory" package to cover all features.
```

Comment:
I think you want the add the blurb here about Victory being an optional peer dependency.  (It is mentioned in the component updates section of release highlights).
This is important to highlight be cause your build will fail without the dependencies.